idea/resources/META-INF/jvm-common.xml

```diff
@@ -138,7 +138,7 @@
                 implementationClass="org.jetbrains.kotlin.idea.inspections.api.IncompatibleAPIInspection"
                 key="inspection.incompatible.API.display.name" bundle="messages.KotlinBundle"/>
 
-        <projectService serviceInterface="org.jetbrains.uast.kotlin.BaseKotlinUastResolveProviderService"
+        <applicationService serviceInterface="org.jetbrains.uast.kotlin.BaseKotlinUastResolveProviderService"
                         serviceImplementation="org.jetbrains.uast.kotlin.internal.IdeaKotlinUastResolveProviderService"/>
         <projectService serviceInterface="org.jetbrains.uast.kotlin.KotlinUastResolveProviderService"
```

Comment:
Maybe, the questions are,
* why was this set as a project service in the beginning? and
* should we make it an application service too?

Reply:
@xiexed